src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -50,6 +45,11 @@ class Feedback extends React.Component {
     return skillObj;
   };
 
+  state = {
```

Comment:
Please put state declaration in this case inside the constructor.

Reply:
okay